src/pkg/packaging/deb/package.targets

```diff
@@ -283,6 +284,88 @@
           UseHardlinksIfPossible="False" />
 
   </Target>
+  
+  <Target Name="GenerateRuntimeMetaDeb">
```

Comment:
This may just be a personal preference, but I don't really like the name "MetaDeb".  It doesn't really tell me what this package is. Can this be renamed to "RuntimeDependenciesDeb"?  That gives the reader a better indication of what this thing is.